### DISCUSSION TITLE
Installation process

### DISCUSSION THREAD

**Discussion by asetmumin99** [pilotmoon/Scroll-Reverser#117] [Q&A] [answered]
I'm new to mac os and recently bought a macbook with m1.
I found that there is no guidance in the readme about the actual installation process, unless I'm blind, of course.

After I've cloned the repo, what should I do? 
Thanks for your time and effort.

**Comment by pilotmoon** [comment, marked as answer]
Hi. If you would like to simply use Scroll Reverser, download the zip file of the [latest release](https://github.com/pilotmoon/Scroll-Reverser/releases/latest).

You can also find this on the product web page at https://pilotmoon.com/scrollreverser/.

Then unzip it, pop it in your `/Applications` folder, and run.

If you are looking to build it from source yourself you will need Xcode. Happy to help with any more detailed queries.

**Comment by Strangehill** [comment]
Also can download the unzipped version from pilotmoon.com and move that to the `/Applications` folder.
